src/service/resources-impl.js

```diff
@@ -1001,7 +1010,7 @@ export class Resources {
     // Phase 3: Trigger "viewport enter/exit" events.
     for (let i = 0; i < this.resources_.length; i++) {
       const r = this.resources_[i];
-      if (r.hasOwner()) {
+      if (r.getState() == ResourceState.NOT_BUILT || r.hasOwner()) {
```

Comment:
Can't make a comment below, but I believe we also need to check the build status in the following code below:

`if (r.getState() != ResourceState.READY_FOR_LAYOUT || r.hasOwner()) {`